spec/services/diagnostics/move_inspector_spec.rb

```diff
@@ -5,28 +5,46 @@
 
   let(:person) { create(:person) }
   let(:profile) { create(:profile, person: person) }
-  let(:move) { create(:move, :in_transit, :court_appearance, profile: profile) }
+  let(:move) { create(:move, :in_transit, :court_appearance, profile: profile, from_location: sch) }
+  let(:person_escort_record) { create(:person_escort_record, profile: profile, move: move) }
+  let(:youth_risk_assessment) { create(:youth_risk_assessment, profile: profile, move: move) }
   let(:journey) { create(:journey, move: move) }
   let(:include_person_details) { false }
   let(:journey_event) { create(:event_journey_start, eventable: journey) }
   let(:move_event) { create(:event_move_start, eventable: move) }
+  let(:sch) { create(:location, :sch) }
+  let(:webhook) { create(:notification, :webhook, topic: move) }
+  let(:email) { create(:notification, :email, topic: person_escort_record) }
 
   before do
-    journey       # create the journey
-    journey_event # create the journey event
-    move_event    # create the move event
+    move
+    move_event
+    journey
+    journey_event
+    person
+    profile
+    person_escort_record
+    youth_risk_assessment
+    webhook
+    email
```

Comment:
You could remove this block completely by changing each of the relevant `let(:foo)` statements above to `let!(:foo)` (i.e. without the lazy loading). Opinions vary on this but I think for this case it would look cleaner.

Reply:
hmm... (some) previous developers on this project have felt that `let!` wasn't clear enough, and having an explicit `before` block made tests more readible.  Each to his own  🤷

Reply:
It's definitely a marmite thing, feel free to leave as is 😄